components/load_from_hf_hub/src/main.py

```diff
@@ -13,52 +11,43 @@
 logger = logging.getLogger(__name__)
 
 
-def extract_width(image_bytes):
-    # Decode image bytes to PIL Image object
-    pil_image = Image.open(io.BytesIO(image_bytes))
-    width = pil_image.size[0]
-
-    return np.int16(width)
-
-
-def extract_height(image_bytes):
-    # Decode image bytes to PIL Image object
-    pil_image = Image.open(io.BytesIO(image_bytes))
-    height = pil_image.size[1]
-
-    return np.int16(height)
-
-
 class LoadFromHubComponent(LoadComponent):
-    def load(self, *, dataset_name: str) -> dd.DataFrame:
+    def load(self,
+             *,
+             dataset_name: str,
```

Comment:
```suggestion
             dataset_name: str,
             *,
```

The `dataset_name` is the main argument and the subject to `load`, so it makes sense to include this as a positional argument.

This makes sense:
```python
component.load("my_dataset")
```

This doesn't:
```python
component.load("my_dataset", {"original_name": "new_name"}, ["original_name"], 5)
```